token/js/cli/token-test.js

```diff
@@ -242,6 +311,26 @@ export async function transferChecked(): Promise<void> {
   assert(testAccountInfo.amount.toNumber() === 1800);
 }
 
+export async function transferCheckedAssociated(): Promise<void> {
+  const dest = new Account().publicKey;
+  let associatedAccount;
+
+  associatedAccount = await testToken.getOrCreateAssociatedAccountInfo(dest);
+  assert(associatedAccount.amount.toNumber() === 0);
+
+  await testToken.transferChecked(
+    testAccount,
+    associatedAccount.address,
+    testAccountOwner,
+    [],
+    123,
+    testTokenDecimals,
+  );
+
+  associatedAccount = await testToken.getOrCreateAssociatedAccountInfo(dest);
+  assert(associatedAccount.amount.toNumber() === 123);
```

Comment:
thanks to https://github.com/solana-labs/solana-program-library/pull/1364/files#r586222243, this is so simple for api users. :)